cmd/archive.go

```diff
@@ -68,6 +75,8 @@ func (c *cmdArchive) flagSet() *pflag.FlagSet {
 	flags.AddFlagSet(optionFlagSet())
 	flags.AddFlagSet(runtimeOptionFlagSet(false))
 	flags.StringVarP(&c.archiveOut, "archive-out", "O", c.archiveOut, "archive output filename")
+	flags.BoolVarP(&c.skipEnv, "skip-env", "", false, "exclude from the archive environment variables passed with `--env`")
```

Comment:
hmm I am not sure this is the best name :thinking: though I am not sure the alternatives I can think of are any better :disappointed:  `--without-env` or maybe `--exclude-env`?

```suggestion
	flags.BoolVarP(&c.excludeEnv, "exclude-env", "", false, "do not embed any environment variables (either from --env or the actual environment) in the archive metadata")
```

since these env vars will be available in the script init phase, they just won't be embedded in the archive, it's best for the flag name to also indicate that, right?

Reply:
>or the actual environment

But as far as I know, we never include actual environment variable values into the archives, right? :thinking:

Reply:
we can, if the user executes `k6 archive --include-system-env-vars=true script.js`, see https://k6.io/docs/using-k6/k6-options/reference#include-system-env-vars

Reply:
True :+1: